README.md

```diff
@@ -185,21 +185,31 @@ yarn run start
 
 ### Verifying the documentation
 
-We have a built-in tool to validate `.md` files called [Sclicheck](/sclicheck/Readme.md). To check all douments (and this is what we run on CI) run:
-
-```.github/scripts/check_docs.sh```
-
-You can also check single documents or directories using
-
-
+We have a built-in tool to validate `.md` files called [Sclicheck](/sclicheck/Readme.md). 
+All `Sclicheck` tests can be run with `Mill` + `munit`: (and this is what we run on the CI, too)
```

Comment:
I think it should describe what kind of `tests` is run.   


What is the difference between running `./mill -i docs-tests.test` and `/mill -i docs-tests.test 'sclicheck.DocTests.*'`. I suppose, that the first command also checks all documents.

Reply:
hm... the former also tests gifs and the tool itself.
I added an extra line of explanation.